robottelo/ui/navigator.py

```diff
@@ -20,7 +21,14 @@ def menu_click(self, top_menu_locator, sub_menu_locator,
                 "arguments[0].click();",
                 tertiary_element,
             )
-        self.browser.refresh()
+        # 1328627 - as the number of subs increases, dashboard page gets slower
+        wait_timeout = 12
+        if bz_bug_is_open(1328627):
```

Comment:
Fair point - here's the answer:
https://github.com/SatelliteQE/robottelo/blob/master/robottelo/decorators.py#L252
tl;dr - no, it uses a cache, so every fetched BZ is saved in the `_bugzilla` dict for later use,